DoViBaker/DoViBaker.cpp

```diff
@@ -1,8 +1,24 @@
-#include "DoViBaker.h"
-#include "cube.h"
+#include "../include/DoViBaker.h"
+#include "../include/cube.h"
 
 #include <array>
-#include <io.h>
+#ifdef _WIN32
+	#include <io.h>
+#elif __linux__
+	#include <inttypes.h>
+	#include <stdlib.h>
+	#include <unistd.h>
+	#include <algorithm>
+	#include <bits/stdc++.h>
+	#define _aligned_malloc(size, alignment) aligned_alloc(alignment, size)
+	#ifndef min
+		#define min(a,b) (((a) < (b)) ? (a) : (b))
+	#endif
+
+	#ifndef max
+		#define max(a,b) (((a) > (b)) ? (a) : (b))
+	#endif
```

Comment:
Not needed. It's better to use std::max/min so all max/min in the file should be changed to std::max/min.